<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>703</width>
    <height>458</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2222</width>
    <height>2222</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>新宋体</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>581</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Device Parameter</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>541</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Init CAN</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>501</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_acc">
        <property name="text">
         <string>Acc Code:0x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_acc">
        <property name="text">
         <string>00000000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_mask">
        <property name="text">
         <string>Mask Code:0x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_mask">
        <property name="text">
         <string>ffffffff</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_filter">
        <property name="text">
         <string>Filter:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_filter">
        <item>
         <property name="text">
          <string>Dual Filter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Single Filter</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>491</width>
       <height>49</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_btr0">
        <property name="text">
         <string>BTR0:0x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_btr0">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_btr1">
        <property name="text">
         <string>BTR1:0x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_btr1">
        <property name="text">
         <string>14</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_mode">
        <property name="text">
         <string>Mode:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_mode">
        <item>
         <property name="text">
          <string>Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ReadOnly</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="BtnConnect">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_dev">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>12</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>新宋体</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Device:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_dev">
       <item>
        <property name="text">
         <string>USBCAN-2I/2I+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USBalyst-II</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PCICAN-9820</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PCICAN-9840</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>iCANEC-I</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>104plusCAN-II</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USBCAN-I</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PCIE9221</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_index">
       <property name="text">
        <string>Index:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_index">
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_can">
       <property name="text">
        <string>CAN:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_can">
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>691</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Send Data Frame</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>673</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_send">
       <property name="text">
        <string>Send Format:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_send">
       <item>
        <property name="text">
         <string>Normal Send</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Single Send</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Send &amp; Recive</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Single Send &amp; Recive</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_type">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_ComboSendFrmType">
       <item>
        <property name="text">
         <string>Extern Frame</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Standard Frame</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_format">
       <property name="text">
        <string>Format:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_format">
       <item>
        <property name="text">
         <string>Data Frame</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Remote Frame</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_ID">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_EditSendFrmID">
       <property name="text">
        <string>00000080</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_data">
       <property name="text">
        <string>Data:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_EditSendData">
       <property name="text">
        <string>01 02 03 04 05 06 07 08</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnSend">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>1221</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>information</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1201</width>
      <height>141</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>0</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>200</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>0</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>158</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="BtnStart">
      <property name="text">
       <string>Start CAN</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnReset">
      <property name="text">
       <string>Reset CAN</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCustomPlot" name="curve" native="true">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>389</y>
     <width>1181</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
